src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/Inputs/SARIF2004.OptimizeFileSize_Valid.sarif

```diff
@@ -42,6 +42,24 @@
               }
             }
           ]
+        },
+        {
+          "rule": {
+            "id": "TEST1001",
+            "toolComponent": { "name": "Plugin01" }
+          },
+          "message": { "text": "Good result: artifacts array provides additional useful information." }
```

Comment:
>Good result: artifacts array provides additional useful information [](start = 32, length = 67)

This comment isn't right. The reason this is valid is that `rule.toolComponent` is present.